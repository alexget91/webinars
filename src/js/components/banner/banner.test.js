import React from "react";
import renderer from "react-test-renderer";
import {MemoryRouter} from "react-router-dom";
import Banner from "./banner";

it(`Banner correctly renders after relaunch`, () => {
  const tree = renderer
    .create(<MemoryRouter>
      <Banner
        title={`title`}
        srcBig={`srcBig`}
        srcSmall={`srcSmall`}
        url={`url`}
        cssClass={`cssClass`}
      />
    </MemoryRouter>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
