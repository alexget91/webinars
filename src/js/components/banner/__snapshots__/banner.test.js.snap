// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner correctly renders after relaunch 1`] = `
<div
  className="card card--double cssClass"
>
  <a
    className="card__double-inner"
    href="/url"
    onClick={[Function]}
  >
    <div
      className="catalog__mobile-hidden"
    >
      <img
        alt="title"
        src="srcBig"
      />
    </div>
    <div
      className="catalog__desktop-hidden"
    >
      <img
        alt="title"
        src="srcSmall"
      />
    </div>
  </a>
</div>
`;
